canopen/pdo/__init__.py

```diff
@@ -4,7 +4,7 @@
 from canopen.pdo.base import PdoBase, Maps
 
 # Compatibility
-from .base import Variable
+from canopen.pdo.base import Variable
```

Comment:
Why change to absolute package reference here?  I'd actually rather go the other way and use relative imports within the same package.  Maybe it's a matter of taste, but I liked the old style.

Reply:
No other reason that PEP 8 recommends using absolute imports, so it's become a habit for many projects. PEP 8 also states "explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose". So either is acceptable, but with preference to the former. And lastly, the _rest_ of canopen is using absolute imports. I find it messy that the import styles are mixed.